fe/fe-core/src/main/java/org/apache/doris/analysis/IndexDef.java

```diff
@@ -34,6 +35,7 @@ public class IndexDef {
     private String indexName;
     private boolean ifNotExists;
     private List<String> columns;
+    private List<String> caseSensitivityColumns = Lists.newArrayList();
```

Comment:
Please add some comments for the purpose.

Reply:
update